### PR TITLE
add description of future modelling tool

### PR DESCRIPTION
**Pull request by derhuerst**
@dirkschumacher

### CODE REVIEW

**Review by juliuste — APPROVED** [review]

**Comment by dirkschumacher** [comment]
I approve. Please do the needful.

File: readme.md [hunk 1]
```diff
@@ -69,3 +69,11 @@ TODO: map picture
 - interactive visualisations
 	- hide less important lines to reduce noise, show them on zoom
 	- route planning, similar to [vbb-map-routing](https://github.com/derhuerst/vbb-map-routing)
+
+## outlook: interactive modelling tool
+
+We may create an interactive transit map modelling tool.
```
Comment (dirkschumacher) [inline readme.md:75]: We could write something along: "the goal is assist manual editing with an automated optimization approach to find initial solutions or improve existing ones."

Comment (derhuerst) [reply readme.md:75]: sounds good to me. feel free to do so!

**Comment by derhuerst** [comment]
@dirkschumacher go ahead if you want to add something.

**Pull request merged**